### ISSUE TITLE
Uncaught TypeError: Cannot read property 'appendChild' of null

### ISSUE THREAD

**Issue by JuanCrg90** [dmuy/duDatepicker#10]
Hello I've trying to use the last datepicker versión in a rails application, but when I try to call the initializer I get the following error:

```
document.ready(function {
Uncaught TypeError: Cannot read property 'appendChild' of null
    at Object.appendTo (duDatepicker.js:69)
    at _duDatePicker._init (duDatepicker.js:966)
    at new _duDatePicker (duDatepicker.js:884)
    at duDatepicker.js:1822
    at Array.forEach (<anonymous>)
    at duDatepicker (duDatepicker.js:1820)
    at HTMLDocument.<anonymous> (datepicker.js:7)
```

This is my initializer:

```js
import '@dmuy/datepicker/dist/duDatepicker.css'
import duDatepicker from '@dmuy/datepicker'

document.addEventListener("DOMContentLoaded", function(){
  duDatepicker('#appointment_form_presenter_date', { format: 'mm-dd-yyy' });
})

```

I took the snippet from the wiki https://github.com/dmuy/duDatepicker/wiki/Installation

I'm not sure If I'm missing something else.

I hope you could help me. Thanks in advance 😄

**Comment by dmuy** [comment]
Hi @JuanCrg90,

I'm not actually familiar on rails, and if it affects DOM manipulation. Also, I tried initializing a date picker using `DOMContentLoaded` event on a static HTML and it's working fine.

Btw, what browser did you try this on?

**Comment by JuanCrg90** [comment]
I tried on Google chrome.

Rails uses web we pack to manage the js dependencies.

I will try to load the dependency from the CDN and I'll add more information.

**Comment by dmuy** [comment]
Hi @JuanCrg90,

I've tracked the cause to the `root` config default and the order or loading the js file. This occurs if the script is loaded at the top because the `body` is not loaded yet (default `root` config uses `document.body`).

Thanks for reporting this, I've already published the fix (v2.0.1-rc on github & npm).

**Comment by JuanCrg90** [comment]
It's working now, thanks for your help 😃
